# Coordinate frame definitions
camera_frame_id: camera_link
world_frame_id: world
robot_base_frame_id: robot_base
robot_center_frame_id: robot_center
launcher_frame_id: launcher

# Geometric fixed frame offsets
geometry/camera_x_offset: 0.133 # X offset from robot center to camera frame
geometry/camera_z_offset: 0.127 # Z offset from robot center to camera frame
geometry/launcher_z_offset: 0.2765 # Z offset from robot center to launcher frame 
